src/form/ChestAPI/session/network/handler/ClosurePlayerNetworkHandler.php

```
@@ -0,0 +1,22 @@
+
+namespace pocketmine\form\ChestAPI\session\network\handler;
+
+use Closure;
+use pocketmine\form\ChestAPI\session\network\NetworkStackLatencyEntry;
+
+final class ClosurePlayerNetworkHandler implements PlayerNetworkHandler{
+
+	/**
+	 * @param Closure(Closure) : NetworkStackLatencyEntry $creator
+	 */
+	public function __construct(
```

PHPStan (2.1.15)

Method pocketmine\form\ChestAPI\session\network\handler\ClosurePlayerNetworkHandler::__construct() has parameter $creator with no signature specified for Closure.


(missingType.callable)